src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SmiEventSink_Default.netcore.cs

```diff
@@ -6,32 +6,17 @@
 
 namespace Microsoft.Data.SqlClient.Server
 {
-    internal class SmiEventSink_Default : SmiEventSink
+    internal partial class SmiEventSink_Default : SmiEventSink
     {
-        private SqlErrorCollection _errors;
-        private SqlErrorCollection _warnings;
-
-
         internal bool HasMessages
```

Comment:
Avoid the code redundancy even just a function definition. All the methods in this file exist on both projects and can be merged.